src/api-client.ts

```diff
@@ -54,7 +58,8 @@ export class CortexApiClient {
       throw new Error("Request body too large");
     }
 
-    return fetch(`${this.apiUrl}/org/${this.org}${path}`, {
+    const url = Path.join(this.apiUrl, this.apiVersion, "org", this.org, path);
```

Comment:
`Path.join` uses a platform specific delimiter, which could be `\` on windows-based machines. I do not believe that we want this.

Reply:
Ah that's a good point - the SDK could be running on Windows, unlike our App or server-side code. Will go back to interpolated string

Reply:
Actually, found a way to use forward slashes consistently: `Path.posix.join`